addons/mail/models/mail_channel.py

```diff
@@ -330,7 +330,7 @@ def _action_unfollow(self, partner):
         # channel_info is called before actually unpinning the channel
         channel_info['is_pinned'] = False
         self.env['bus.bus']._sendone(partner, 'mail.channel/leave', channel_info)
-        notification = _('<div class="o_mail_notification">left the channel</div>')
+        notification = _('<div class="o_mail_notification">%(member)s left the channel</div>', member=member.partner_id.name or member.guest_id.name)
```

Comment:
While we are at it, the HTML should be outside of the translated part. Same remark for all others.